cloud/azure/management/applicationinsights/mode/cpu.pm

```diff
@@ -0,0 +1,151 @@
+#
+# Copyright 2021 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package cloud::azure::management::applicationinsights::mode::cpu;
+
+use base qw(cloud::azure::custom::mode);
+
+use strict;
+use warnings;
+
+sub get_metrics_mapping {
+    my ($self, %options) = @_;
+
+    my $metrics_mapping = {
+        'performanceCounters/processCpuPercentage' => {
+            'output' => 'Process CPU',
+            'label'  => 'process',
+            'nlabel' => 'appinsights.cpu.w3wp.utilization.percentage',
+            'unit'   => '%',
+            'min'    => '0',
+            'max'    => '100'
+        },
+        'performanceCounters/processorCpuPercentage' => {
+            'output' => 'Processor time',
+            'label'  => 'processor-time',
+            'nlabel' => 'appinsights.cpu.utilization.percentage',
```

Comment:
first metric: Is 'process' always refers to an IIS process? if not, maybe it could be process instead of w3wp. Also, percentage in threshold label could help to not misunderstanding the option purpose. 
second metric: add processor in nlabel to be consistent. Also, why using time instead of percentage on this one?

Reply:
The description of the metric says "This metric indicates the performance of w3wp process alone"

Reply:
The second one is the percentage of time the processor spends in non-idle threads. I'll try to improve the labels